tests/e2e/testRunner.ts

```diff
@@ -137,10 +144,33 @@ const runTests = async (): Promise<void> => {
         Logger.log('Launching', appPackage);
         await launchApp('android', appPackage, config.ACTIVITY_PATH, launchArgs);
 
+        start(appPackage);
```

Comment:
NIT: when I read this code I don't understand what "start" does - can we either wildcard import it to be like MeasureUtils.start(), or rename the function to make this more clear?